weapons/allied-infantry.yaml

```diff
@@ -294,3 +294,33 @@ awp:
 	Warhead@3EffWater: CreateEffect
 		Explosions: piff
 		ValidImpactTypes: Water
+
+NeutronRifle:
+	ValidTargets: Ground
+	ReloadDelay: 120
+	Range: 10c0
+	MinRange: 0c001
+	Report: ichratta.wav
+	Projectile: LaserZap
```

Comment:
A TeslaZap DimZap: 1 BrightZap :0 using a custom RGBA palette is closer what RA2 had imo.